pkg/machinepool/machinepool.go

```diff
@@ -18,6 +21,7 @@ var notFoundMessage string = "Machine pool '%s' not found"
 type MachinePoolService interface {
 	DescribeMachinePool(r *rosa.Runtime, cluster *cmv1.Cluster, clusterKey string, isHypershift bool,
 		machinePoolId string) error
+	ListMachinePools(r *rosa.Runtime, clusterKey string, cluster *cmv1.Cluster, isHypershift bool)
```

Comment:
Can we update this func to return an error and we leave it up to the cmd to print and exit. Will make this func a lot easier to test too